### DISCUSSION TITLE
TorchIO tutorial.ipynb

### DISCUSSION THREAD

**Discussion by idhamari** [TorchIO-project/torchio#1188] [General]
Thank you for your effort. The [tutorial](https://colab.research.google.com/github/fepegar/torchio-notebooks/blob/main/notebooks/TorchIO_tutorial.ipynb) is simple, clear and run without issue. 
Could you please add a section to the tutorial that uses the model in production e.g. 

1. Reading a single image and transforming it (without using dataset class). 
2. Applying the inference using a pre-trained model to get a result image and save it with the original size. 
3. Compute the dice of the original label (without resizing) and the saved output from step2. 

I think there will be an issue as the used transforms do not preserve the spacing and directions.
